lib/bulk_data/government_repository.rb

```diff
@@ -46,5 +46,13 @@ def populate_cache(older_than: nil)
     rescue GdsApi::BaseError
       raise RemoteDataUnavailableError
     end
+
+    def age_of_cache
+      (Time.zone.now - Cache.written_at(CACHE_KEY)).seconds
```

Comment:
Time.current is preferred to using Time.zone.now, I wonder if I can make that a linter rule as it comes up a bit 🤔 

I'm not sure we need the `.seconds` as I believe this will execute the same with it or without it.

Reply:
Absolutely right on the `.seconds`. I'm happy to do the linter rule. Do you think that needs doing just in content publisher or in https://github.com/alphagov/rubocop-govuk?